articles/azure-monitor/app/worker-service.md

```diff
@@ -217,13 +217,13 @@ The full example is shared at this [GitHub page](https://github.com/microsoft/Ap
     ```
 
 1. Set up the connection string.
-   Use the same `appsettings.json` from the preceding .NET Core [LTS](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) Worker Service example.
+   Use the same `appsettings.json` from the preceding [.NET Core](https://learn.microsoft.com/dotnet/fundamentals/) Worker Service example.
 
 ## .NET Core/.NET Framework console application
 
-As mentioned in the beginning of this article, the new package can be used to enable Application Insights telemetry from even a regular console application. This package targets [`NetStandard2.0`](/dotnet/standard/net-standard), so it can be used for console apps in .NET Core [LTS](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) or higher, and .NET Framework [LTS](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) or higher.
+As mentioned in the beginning of this article, the new package can be used to enable Application Insights telemetry from even a regular console application. This package targets [`NetStandard2.0`](/dotnet/standard/net-standard), so it can be used for console apps in [.NET Core](https://learn.microsoft.com/dotnet/fundamentals/) or higher, and [.NET Framework](https://learn.microsoft.com/dotnet/framework/) or higher.
```

Comment:
```suggestion
As mentioned in the beginning of this article, the new package can be used to enable Application Insights telemetry from even a regular console application. This package targets [`.NET Standard 2.0`](/dotnet/standard/net-standard), so it can be used for console apps in [.NET Core](https://learn.microsoft.com/dotnet/fundamentals/) or higher, and [.NET Framework](https://learn.microsoft.com/dotnet/framework/) or higher.
```
don't know if you want to fix any of those ^^

Reply:
fixed